### DISCUSSION TITLE
[Vote ended on 2023-08-22] Remove community.azure from Ansible 10

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#264] [Votes done]
[community.azure](https://github.com/ansible-collections/community.azure) is deprecated: ansible-collections/community.azure#26

It's officially unmaintained and archived, so I propose to remove it from Ansible 10.

Please vote with +1 to remove it from Ansible 10 or -1 to keep it.

If you want to discuss this, please use #263

**Comment by mariolenz** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by maxamillion** [comment]
+1

**Comment by leogallego** [comment]
+1

**Comment by cybette** [comment]
+1
